src/electionguard_cli/e2e/election_record_step.py

```diff
@@ -30,7 +29,7 @@ def export(
         self._export_election_record(
             election_inputs, build_election_results, submit_results, decrypt_results
         )
-        self._export_private_keys(election_inputs)
+        self._export_private_keys_e2e(election_inputs)
```

Comment:
The answer generally is because I'm an adherent of Clean Code by Bob Martin, in which he argues for small methods with as few parameters as possible.  The answer specifically is because the class `ElectionRecordStep` is a class that's specific to the E2E command (I renamed it to make that clearer in the next PR), and so it feels reasonable to accept a single parameter that is also specific to the E2E command as a convenience method that then calls the more generic parent class.  However, I don't feel particularly strongly about it in this instance and would be happy to delete the convenience method if you would prefer now that you understand my reasoning.